src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,53 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[];
+  onTabSelect: (tab: Tab) => void;
+  selectedTabId: string;
+}
+
+export const Tabs: React.FC<Props> = ({ tabs, onTabSelect, selectedTabId }) => {
+  const currentTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const handleTabClick = (tab: Tab) => {
+    if (tab.id !== currentTab.id) {
+      onTabSelect(tab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              data-cy="Tab"
+              className={classNames(
+                {
+                  'is-active': tab.title === currentTab.title,
+                },
+              )}
+            >
+              <a
+                href={tab.id}
```

Comment:
![image](https://github.com/mate-academy/react_tabs/assets/91411692/e5ffa2b8-0c12-4af2-8d20-c83f0c18b3f1)

```suggestion
                href={`#${tab.id}}
```